DataFormats/HGCalReco/interface/Trackster.h

```diff
@@ -153,7 +153,7 @@ namespace ticl {
     // The vertices of the DAG are the indices of the
     // 2d objects in the global collection
     std::vector<unsigned int> vertices_;
-    std::vector<uint8_t> vertex_multiplicity_;
+    std::vector<float> vertex_multiplicity_;
```

Comment:
I'm curious if the schema evolution handles this appropriately.
@felicepantaleo please clarify/confirm that tracksters from the old files are still correctly readable.

Reply:
I could run the trackster validation on old files with this branch.